### DISCUSSION TITLE
IDEA: MARQUEE

### DISCUSSION THREAD

**Discussion by AMagnitude67Earthquake** [DerpyAxolotl/derpyaxolotl.github.io#3] [Ideas]
to add improvement, make the addition of a marquee. please discuss.

**Comment by AMagnitude67Earthquake** [comment]
please create a response.

**Reply by AMagnitude67Earthquake** [reply]
> please look at this.

**Reply by DerpyAxolotl** [reply]
ok

**Comment by AMagnitude67Earthquake** [comment]
i will continue to add comments here.

**Reply by AMagnitude67Earthquake** [reply]
replys also.

**Reply by AMagnitude67Earthquake** [reply]
can you reply to replys.

**Reply by AMagnitude67Earthquake** [reply]
seemingly no.

**Reply by AMagnitude67Earthquake** [reply]
what sadness.

**Reply by AMagnitude67Earthquake** [reply]
> i will continue to add comments here.

quotes exist. yay.

**Reply by AMagnitude67Earthquake** [reply]
> > i will continue to add comments here.
> 
> quotes exist. yay.

and you can quote quotes.

**Reply by AMagnitude67Earthquake** [reply]
> > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > how much quoting can be done.

**Reply by AMagnitude67Earthquake** [reply]
> > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > how much quoting can be done.

very much it seems. you don.t even have to quote actual things. fun.

**Reply by AMagnitude67Earthquake** [reply]
> > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > quote.

**Reply by AMagnitude67Earthquake** [reply]
> it seems as though quotes can overflow. interesting.

**Reply by AMagnitude67Earthquake** [reply]
> > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > there is not much point to reading this.

**Reply by AMagnitude67Earthquake** [reply]
> it seems as though at a certain point they stop appearing. how unhelpful.

**Comment by AMagnitude67Earthquake** [comment]
> new comment.

**Reply by AMagnitude67Earthquake** [reply]
> 🚀

**Comment by DerpyAxolotl** [comment]
>You're stupid-Edward Wu
